src/common/AndroidUtils.ts

```diff
@@ -510,45 +510,57 @@ export class AndroidUtils {
                 return AndroidUtils.emulatorHasPort(resolvedEmulator);
             })
             .then(async (port) => {
-                const resolvedPortNumer = port
+                const resolvedPortNumber = port
                     ? port
                     : await AndroidUtils.getNextAvailableAdbPort();
 
-                if (resolvedPortNumer === port) {
+                if (resolvedPortNumber === port) {
                     // already is running on a port
                     const isWritable = await AndroidUtils.isEmulatorSystemWritable(
-                        resolvedPortNumer
+                        resolvedPortNumber
                     );
 
                     if (writable === false || isWritable === true) {
                         // If we're not asked for a writable, or if it already
                         // is writable then we're done so just return its port.
-                        return Promise.resolve(resolvedPortNumer);
+                        return Promise.resolve(resolvedPortNumber);
                     }
 
                     // mismatch... shut it down and relaunch it in the right mode
-                    await AndroidUtils.stopEmulator(resolvedPortNumer);
+                    CommonUtils.updateCliAction(
```

Comment:
leaving these as hard-coded strings for now b/c we have it that way in all 3 repos currently and have a separate story about cleaning that up.